master/pkg/tasks/task.go

```diff
@@ -281,6 +282,18 @@ func gcCheckpoint(t TaskSpec) container.Spec {
 				workDirArchive(gcc.AgentUserGroup),
 				wrapArchive(
 					archive.Archive{
+						gcc.AgentUserGroup.OwnedArchiveItem(
+							"experiment_config.json",
+							[]byte(jsonify(gcc.ExperimentConfig)),
+							0600,
+							tar.TypeReg,
+						),
+						gcc.AgentUserGroup.OwnedArchiveItem(
+							"checkpoints_to_delete.json",
+							[]byte(jsonify(gcc.ExperimentConfig)),
```

Comment:
:thinking:

Reply:
jsonify returns a string, but we need a byte array.  Is there a better way?

Reply:
```suggestion
							[]byte(jsonify(gcc.ToDelete)),
```
:wink:

(I'm not generally the sort to use emoji to communicate, but it seems to fit here.)

Reply:
ah, I see

Reply:
I can confirm that the test is not working.  I'll also fix the test while I'm at it.

Good catch here.

Reply:
We still pretty much just throw out all the results of checkpoint GC. At least the logs are showing up in the master now, but the experiment still completes successfully, which is usually the only thing that gets checked in tests. I'll file a ticket about checking GC failures.

Reply:
Whoops, you posted your comment while I was writing mine. Didn't realize we actually did have a test already. I won't bother with the ticket, then.